### PR TITLE
docs(textarea): add ux description for textarea

### PR DESCRIPTION
**Pull request by flxlst09**
## 🆕 What is the new behavior?

New UX description for textarea

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [ ] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [ ] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [ ] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [ ] 🧐 Static code analysis passes (`pnpm lint`)
- [ ] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:1]: We normally always add a “We typically use...” to the intro sentence. Like: The textarea component allows users to input multi-line text, making it ideal for forms that require longer text entries. We typically use textareas in scenarios such as feedback forms, comment sections, and message composition.

Also please pay attention to capitalization “textarea” vs “Textarea”.

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+![Textarea overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3814-1128&t=DtCmoFcLwhf7ke3S-4)
+1. Label
+2. Required field indicator (optional)
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:5]: Normally, we never specified in anatomy whether something is optional or not. If desired, we would need to consistently maintain this in all Markdowns.

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+3. Placeholder text
+4. Container
+5. Resize handle
+6. Helper or feedback text
+7. Counter
+
+## Options
+
+- **Label**: [See form field](./forms/forms-field.md)
+- **Value**: [See form field](./forms/forms-field.md)
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:15]: The ‘See’ should't be part of the link.

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- **Required**: [See form field](./forms/forms-field.md)
+- **Helper text**: [See form field](./forms/forms-field.md)
+- **Feedback text**: [See form field](./forms/forms-field.md)
+- **Show text as tooltip**: [See form field](./forms/forms-field.md)
+- **Placeholder text**: [See form field](./forms/forms-field.md)
+- **Counter**: [See form field](./forms/forms-field.md)
+- **Resize behavior**: Determines how textareas can be resized (both directions, horizontally, vertically, no resizing)
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:22]: We need to include: what is the standard size, what is our default version, and when do we use each option?

Comment (bettiifr) [reply packages/documentation/docs/controls/_textarea-field_styleguide.mdx:22]: Additionally, is this an option or should it be handled in the behavior in context

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- **Columns and width**: Defines inital width by number of columns and/or width
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:23]: Also here is this an option or should it be handled in the behavior in context. How can I change it, can I change it, and what should be considered from a UX perspective?

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- **Rows and height**: Defines inital height by number of rows and/or height
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:24]: Also here is this an option or should it be handled in the behavior in context. How can I change it, can I change it, and what should be considered from a UX perspective?

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## Behavior
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:27]: Behavior in context

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+- **Interaction**:
+  - Clicking in the container enables the editing of the field
+  - Users can type, copy, paste, and cut text within textareas
+  - Optional: Users can resize textareas to fit their needs
+- **Validation**:
+  - Minimum and maximum length defines number of characters allowed
+  - [See form validation](./forms/forms-validation.md)
+- **Overflow**: Text within the textarea is not truncated; it supports scrolling for overflow content
+- **Alignment**: Left aligned
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:37]: Can the alignment be adjusted

Comment (bettiifr) [reply packages/documentation/docs/controls/_textarea-field_styleguide.mdx:37]: In general, please use periods. Complete sentences should always be used in every section, except in the ‘do and don’t’ section.

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+## States
+
+- Default
+- Hover
+- Focused
+- Read-only
+- Disabled
+
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:46]: Either include the states in a sentence or use only the image.

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+![Textarea states](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3947-527&t=DtCmoFcLwhf7ke3S-4)
+
+## Dos and Don’ts
+
+- Do use the textarea for inputs that require multiple lines of text, such as comments, feedback, or detailed descriptions
+- Do ensure the textarea is appropriately sized for the expected input
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:52]: What is meant by ‘appropriately sized’? It is never explained in the doku

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+- Do use the placeholder attribute to give users an example of the expected input
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:53]: Is a placeholder attribute the placeholder text? If so, don't change the wording

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+- Do use minimum and maximum character limits to handle edge cases
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:54]: Do set minimum and maximum character limits to ensure proper input length

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+- Don’t use the textarea for short, single-line inputs like names or email addresses; use an input field instead
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:55]: Is redundant

Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:55]: Don’t forget cross-referencing

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 14]
```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
+
+![Textarea overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3814-1128&t=DtCmoFcLwhf7ke3S-4)
+1. Label
+2. Required field indicator (optional)
+3. Placeholder text
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:6]: Only "Placeholder"

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+
+## Options
+
+- **Label**: [See form field](./forms/forms-field.md)
+- **Value**: [See form field](./forms/forms-field.md)
+- **Required**: [See form field](./forms/forms-field.md)
+- **Helper text**: [See form field](./forms/forms-field.md)
+- **Feedback text**: [See form field](./forms/forms-field.md)
+- **Show text as tooltip**: [See form field](./forms/forms-field.md)
+- **Placeholder text**: [See form field](./forms/forms-field.md)
```
Comment (bettiifr) [inline packages/documentation/docs/controls/_textarea-field_styleguide.mdx:20]: Only "Placeholder"

File: packages/documentation/docs/controls/_textarea-field_styleguide.mdx (hunk 4 above)
Comment (flxlst09) [reply packages/documentation/docs/controls/_textarea-field_styleguide.mdx]: Changed it to have a short notice on resizing options (both, horiz, vert, none) here and guidance how to use it in the behavior section.

**Pull request merged**